.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: '🐛 Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `whatsapp-web.js` (https://github.com/pedroslopez/whatsapp-web.js).
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of `whatsapp-web.js`'s Discord server: https://discord.gg/wyKybbF
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - `whatsapp-web.js` Issues tab: https://github.com/pedroslopez/whatsapp-web.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+         - `whatsapp-web.js` closed issues tab: https://github.com/pedroslopez/whatsapp-web.js/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
+         - `whatsapp-web.js` Discord server: https://discord.gg/wyKybbF
+
+        The more information you fill in, the better the community can help you.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of the challenge you are running into.
+    validations:
+      required: true
+  - type: input
+    id: link
+    attributes:
+      label: Your Example Website or App
+      description: |
+        Which website or app were you using when the bug happened?
+        Note:
+        - Your bug will may get fixed much faster if we can run your code and it doesn't have dependencies other than the `whatsapp-web.js` npm package / dependency.
+        - To create a shareable code example you can use Stackblitz (https://stackblitz.com/). Please no localhost URLs.
+        - Please read these tips for providing a minimal example: https://stackoverflow.com/help/mcve.
+      placeholder: |
+        e.g. https://stackblitz.com/edit/...... OR Github Repo
+    validations:
+      required: true
```

Comment:
Websites aren't really applicable here... Maybe this section should be replaced with a code sample

Reply:
Thanks - I've updated it; Below is a demo link: 
https://github.com/cliffordfajardo/whatsapp-web.js/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml